ddtrace/contrib/tornado/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+The Tornado integration traces all ``RequestHandler`` defined in a Tornado web application.
+Auto instrumentation is available using the ``trace_app`` function as follows::
+
+    import tornado.web
+    import tornado.gen
+    import tornado.ioloop
+
+    from ddtrace import tracer
+    from ddtrace.contrib.tornado import trace_app
+
+    # create your handlers
+    class MainHandler(tornado.web.RequestHandler):
+        @tornado.gen.coroutine
+        def get(self):
+            self.write("Hello, world")
+
+    # create your application
+    app = tornado.web.Application([
+        (r'/', MainHandler),
+    ])
+
+    # trace your application before the execution
+    trace_app(app, tracer, service='tornado-site')
+
+    # and run it as usual
+    app.listen(8888)
+    tornado.ioloop.IOLoop.current().start()
+
+When a ``RequestHandler`` is hit, a request span is automatically created and attached
+to the current ``request`` object, so that it can be used in the application code::
+
+    class MainHandler(tornado.web.RequestHandler):
+        @tornado.gen.coroutine
+        def get(self):
+            ctx = getattr(self.request, 'datadog_context')
```

Comment:
### About the API 

Is that the "best" way we can offer to get the current `Context`? Is `tracer. get_call_context` not working here?
We could offer a small helper for that (not to expose this kind of "internal")?

### Docs

However, since `Context` is already a behind-the-scene thing, we might want to remove this part not to confuse the user. 
The official standard way is to always use the `tracer`. I checked the doc and besides for `iohttp` we never talk about how to get it.
What do you think? If there is no case where getting the context directly is useful,
we should simplify and not talk about how to get it here (but still explain that we attach it to the `request` object).